mllib/src/main/scala/org/apache/spark/ml/r/LDAWrapper.scala

```diff
@@ -172,6 +187,8 @@ private[r] object LDAWrapper extends MLReadable[LDAWrapper] {
       model,
       ldaModel.logLikelihood(preprocessedData),
       ldaModel.logPerplexity(preprocessedData),
+      trainingLogLikelihood,
+      logPrior,
```

Comment:
LogPrior is calculated based on the serialized topics etc, which are also used by the trainingLikelyhood. But the trainingLikelyhood is the same for both original and loaded model. Let me debug more. It looks like a bug. The original MLLIB implementation doesn't serialize the two parameters as they can be calculated from other saved values. In addition, there is no unit test for comparing the two values, which could be the reason of not catching this issue.